build/llhttp.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const { join, resolve } = require('path')
+const { copyFileSync, rmSync } = require('fs')
+const { execSync } = require('child_process')
+const TMP = require('os').tmpdir()
+
+const REPO = 'git@github.com:dnlup/llhttp.git'
```

Comment:
I would prefer if we used the upstream repository instead and we copy over whatever files we need.

Alternatively, I'd be ok in doing that adds this to llhttp itself.

Reply:
> I would prefer if we used the upstream repository instead and we copy over whatever files we need.

Expanding on https://github.com/nodejs/undici/pull/575/files#r589005221 (and thinking out loud), we need to build the `c` files prior to build the wasm binary (and the source files are the same `llhttp` is using to build the parser), I think moving that out of `llhttp` would require us to duplicate a lot of build logic in here. Like, we would need the `c` src files, the `js` files to build the parser, add the build scripts. 

> Alternatively, I'd be ok in doing that adds this to llhttp itself.

I'll open a PR to `llhttp` and see what happens.

Reply:
Ok, here's the PR:

https://github.com/nodejs/llhttp/pull/93